eXo/eXoAppDelegate.swift

```diff
@@ -340,5 +340,4 @@ struct AppUtility {
         UIDevice.current.setValue(rotateOrientation.rawValue, forKey: "orientation")
         UINavigationController.attemptRotationToDeviceOrientation()
     }
-    
```

Comment:
not needed modification

Reply:
> not needed modification

This just clean of code

Reply:
In fact, it add a modification in a file you didnt modify. So it complexify the PR review and in the future the commit read will more complex.
We should keep the commit as small as possible, and this line removing seems not necessary.
By removing the change, we simplify the commit and make it more readable.